confidential-data-hub/hub/protos/api.proto

```diff
@@ -18,10 +18,22 @@ message GetResourceResponse {
     bytes Resource = 1;
 }
 
+message SecureMountRequest {
+    bytes mountInfo = 1;
```

Comment:
so it is implied that the concrete mount type (s.t. driver type to be called) would be included inside the mountInfo? Thus do we have a common struct format of the parameter?